### PR TITLE
[Chef] add Multi-sensor

### PR DESCRIPTION
**Pull request by erwinpan1**
Create Chef sample app which as contactsensor, lightsensor, occupancysensor, temperaturesensor, pressuresensor, flowsensor, humiditysensor, airqualitysensor, powersource

### CODE REVIEW

**Review by andy31415 — APPROVED** [review]

File: examples/chef/devices/rootnode_contactsensor_lightsensor_occupancysensor_temperaturesensor_pressuresensor_flowsensor_humiditysensor_airqualitysensor_powersource_367e7cea91.matter [hunk 1]
```diff
@@ -0,0 +1,3546 @@
...
+    callback attribute acceptedCommandList;
+    callback attribute attributeList;
+    ram      attribute featureMap default = 0;
+    ram      attribute clusterRevision default = 2;
+
+    handle command Identify;
+    handle command TriggerEffect;
+  }
+
+  server cluster Groups {
```
Comment (tcarmelveilleux) [inline examples/chef/devices/rootnode_contactsensor_lightsensor_occupancysensor_temperaturesensor_pressuresensor_flowsensor_humiditysensor_airqualitysensor_powersource_367e7cea91.matter:3213]: Remove groups, not needed

File: examples/chef/devices/rootnode_contactsensor_lightsensor_occupancysensor_temperaturesensor_pressuresensor_flowsensor_humiditysensor_airqualitysensor_powersource_367e7cea91.matter [hunk 2]
```diff
@@ -0,0 +1,3546 @@
...
+    callback attribute acceptedCommandList;
+    callback attribute attributeList;
+    ram      attribute featureMap default = 0;
+    ram      attribute clusterRevision default = 4;
+
+    handle command Identify;
+    handle command TriggerEffect;
+  }
+
+  server cluster Groups {
```
Comment (tcarmelveilleux) [inline examples/chef/devices/rootnode_contactsensor_lightsensor_occupancysensor_temperaturesensor_pressuresensor_flowsensor_humiditysensor_airqualitysensor_powersource_367e7cea91.matter:3275]: Remove group everywhere

File: examples/chef/devices/rootnode_contactsensor_lightsensor_occupancysensor_temperaturesensor_pressuresensor_flowsensor_humiditysensor_airqualitysensor_powersource_367e7cea91.matter [hunk 3]
```diff
@@ -0,0 +1,3546 @@
...
+    callback attribute clientList;
+    callback attribute partsList;
+    callback attribute generatedCommandList;
+    callback attribute acceptedCommandList;
+    callback attribute attributeList;
+    callback attribute featureMap;
+    callback attribute clusterRevision;
+  }
+
+  server cluster OccupancySensing {
```
Comment (tcarmelveilleux) [inline examples/chef/devices/rootnode_contactsensor_lightsensor_occupancysensor_temperaturesensor_pressuresensor_flowsensor_humiditysensor_airqualitysensor_powersource_367e7cea91.matter:3000]: Please run TC_DeviceBasicComposition.py and TC_DeviceConformance.py against the device to resolve issues, and ensure everything is up to latest.

File: examples/chef/devices/rootnode_contactsensor_lightsensor_occupancysensor_temperaturesensor_pressuresensor_flowsensor_humiditysensor_airqualitysensor_powersource_367e7cea91.matter [hunk 4]
```diff
@@ -0,0 +1,3546 @@
...
+    ram      attribute occupancy default = 0;
+    ram      attribute occupancySensorType default = 0;
+    ram      attribute occupancySensorTypeBitmap default = 0x1;
```
Comment (tcarmelveilleux) [inline examples/chef/devices/rootnode_contactsensor_lightsensor_occupancysensor_temperaturesensor_pressuresensor_flowsensor_humiditysensor_airqualitysensor_powersource_367e7cea91.matter:3003]: This is missing the HoldTime/HoldTimeLimits, which are most commonly used and very helpful for clients to support.

File: examples/chef/devices/rootnode_contactsensor_lightsensor_occupancysensor_temperaturesensor_pressuresensor_flowsensor_humiditysensor_airqualitysensor_powersource_367e7cea91.matter (hunk 1 above)
Comment (erwinpan1) [reply examples/chef/devices/rootnode_contactsensor_lightsensor_occupancysensor_temperaturesensor_pressuresensor_flowsensor_humiditysensor_airqualitysensor_powersource_367e7cea91.matter:3213]: Removed.

File: examples/chef/devices/rootnode_contactsensor_lightsensor_occupancysensor_temperaturesensor_pressuresensor_flowsensor_humiditysensor_airqualitysensor_powersource_367e7cea91.matter (hunk 2 above)
Comment (erwinpan1) [reply examples/chef/devices/rootnode_contactsensor_lightsensor_occupancysensor_temperaturesensor_pressuresensor_flowsensor_humiditysensor_airqualitysensor_powersource_367e7cea91.matter:3275]: Removed all Groups clusters

File: examples/chef/devices/rootnode_contactsensor_lightsensor_occupancysensor_temperaturesensor_pressuresensor_flowsensor_humiditysensor_airqualitysensor_powersource_367e7cea91.matter (hunk 3 above)
Comment (erwinpan1) [reply examples/chef/devices/rootnode_contactsensor_lightsensor_occupancysensor_temperaturesensor_pressuresensor_flowsensor_humiditysensor_airqualitysensor_powersource_367e7cea91.matter:3000]: Done

* TC_DeviceBasicComposition.py
```
(py) $ python ./src/python_testing/TC_DeviceBasicComposition.py  --storage-path admin_storage.json --manual-code 34970112332 --bool-arg ignore_in_progress:True allow_provisional:True --PICS src/app/tests/suites/certification/ci-pics-values
...
[MatterTest] 11-12 21:47:08.492 INFO ###########################################################
[MatterTest] 11-12 21:47:08.492 INFO Summary for test class TC_DeviceBasicComposition: Error 0, Executed 8, Failed 0, Passed 8, Requested 14, Skipped 6
...
INFO:root:Final result: PASS !
```

* TC_DeviceConformance.py
```
$ python ./src/python_testing/TC_DeviceConformance.py  --storage-path admin_storage.json --manual-code 34970112332 --bool-arg ignore_in_progress:True allow_provisional:True --PICS src/app/tests/suites/certification/ci-pics-values
...
[MatterTest] 11-13 19:24:17.361 INFO Summary for test class TC_DeviceConformance: Error 0, Executed 3, Failed 0, Passed 3, Requested 3, Skipped 0
[MatterTest] 11-13 19:24:17.362 INFO Summary for test run MatterTest@11-13-2024_19-24-11-654:
Total time elapsed 5.707147184992209s
Artifacts are saved in "/tmp/matter_testing/logs/MatterTest/11-13-2024_19-24-11-654"
Test summary saved in "/tmp/matter_testing/logs/MatterTest/11-13-2024_19-24-11-654/test_summary.yaml"
Test results: Error 0, Executed 3, Failed 0, Passed 3, Requested 3, Skipped 0
...
INFO:root:Final result: PASS !
```

File: examples/chef/devices/rootnode_contactsensor_lightsensor_occupancysensor_temperaturesensor_pressuresensor_flowsensor_humiditysensor_airqualitysensor_powersource_367e7cea91.matter (hunk 4 above)
Comment (tcarmelveilleux) [reply examples/chef/devices/rootnode_contactsensor_lightsensor_occupancysensor_temperaturesensor_pressuresensor_flowsensor_humiditysensor_airqualitysensor_powersource_367e7cea91.matter:2697]: We can do this one separately in a different PR.

**Review by tcarmelveilleux — APPROVED** [review]

**Pull request merged**